aiplatform/src/main/java/aiplatform/imagen/EditImageInpaintingInsertMaskSample.java

```diff
@@ -0,0 +1,129 @@
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Base64;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+public class EditImageInpaintingInsertMaskSample {
+
+  public static void main(String[] args) throws IOException {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "my-project-id";
+    String location = "us-central1";
+    String inputPath = "/path/to/my-input.png";
+    String maskPath = "/path/to/my-mask.png";
+    String prompt =
+        ""; // The text prompt describing what you want to see inserted in the mask area.
+
+    editImageInpaintingInsertMask(projectId, location, inputPath, maskPath, prompt);
+  }
+
+  // Edit an image using a mask file. Inpainting can insert the object designated by the prompt
+  // into the masked area.
+  public static PredictResponse editImageInpaintingInsertMask(
+      String projectId, String location, String inputPath, String maskPath, String prompt)
+      throws ApiException, IOException {
+    final String endpoint = String.format("%s-aiplatform.googleapis.com:443", location);
+    PredictionServiceSettings predictionServiceSettings =
+        PredictionServiceSettings.newBuilder().setEndpoint(endpoint).build();
+
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests.
+    try (PredictionServiceClient predictionServiceClient =
+        PredictionServiceClient.create(predictionServiceSettings)) {
+
+      final EndpointName endpointName =
+          EndpointName.ofProjectLocationPublisherModelName(
+              projectId, location, "google", "imagegeneration@006");
+
+      // Encode image and mask to Base64
+      String imageBase64 =
+          Base64.getEncoder().encodeToString(Files.readAllBytes(Paths.get(inputPath)));
+      String maskBase64 =
+          Base64.getEncoder().encodeToString(Files.readAllBytes(Paths.get(maskPath)));
+
+      // Create the image and image mask maps
+      Map<String, String> imageMap = new HashMap<>();
+      imageMap.put("bytesBase64Encoded", imageBase64);
+
+      Map<String, String> maskMap = new HashMap<>();
+      maskMap.put("bytesBase64Encoded", maskBase64);
+      Map<String, Map> imageMaskMap = new HashMap<>();
+      imageMaskMap.put("image", maskMap);
+
+      Map<String, Object> instancesMap = new HashMap<>();
+      instancesMap.put("prompt", prompt);
+      instancesMap.put("image", imageMap);
+      instancesMap.put("mask", imageMaskMap);
+      instancesMap.put("editMode", "inpainting-insert");
+      Value instances = mapToValue(instancesMap);
+      // instancesMap contents:
+      // [ "image", [ "bytesBase64Encoded", "iVBORw0KGgo...==" ] ]
+      // [ "editMode", "inpainting-insert" ]
+      // [ "prompt", "<my-prompt>" ]
+      // [ "mask", [ "image", [ "bytesBase64Encoded", "iJKDF0KGpl...==" ] ] ]
```

Comment:
nit: would you be able to move these comments to the same line as the code?

For example:

`instancesMap.put("prompt", prompt); // [ "prompt", "<my-prompt>" ]`

Reply:
Done